app/src/core/profile/Profile.ts

```diff
@@ -13,6 +13,8 @@ import { NO_PHOTO } from "../../utils/constants";
 import { currentDate } from "../../utils/datetime";
 import { AVATAR_URL } from "../../utils/files";
 import { getFullName } from "../../utils";
+import CatchErrors from "../CatchErrors";
+import dayjs from "dayjs";
```

Comment:
Please, follow the order of imports. Lines with libraries come first, then local files